packages/core/integration-tests/test/javascript.js

```diff
@@ -717,7 +765,7 @@ describe('javascript', function () {
     assert.equal(await output(), 7);
   });
 
-  it('should not duplicate a module which is already in a parent bundle', async function () {
+  it.skip('should not duplicate a module which is already in a parent bundle', async function () {
```

Comment:
Did you mean to skip these?

Reply:
Nope! Probably some bad remnant, I'll double check for others